T3/Gui/Interaction/SliderLadder.cs

```diff
@@ -56,10 +56,14 @@ public static void Draw(ref double editValue, ImGuiIOPtr io, double min, double
             if (timeSinceVisible < initialDelay)
                 _lockedRange = null;
 
-            var pLast = io.MousePos - io.MouseDelta - center;
-            var pNow = io.MousePos - center;
+            Vector2 pLast = io.MousePos - io.MouseDelta - center;
+            Vector2 pNow = io.MousePos - center;
+            if (!ImGui.IsMousePosValid())
+            {
+                _lastStepPosX = pNow.X = float.NaN;
+            }
 
-            if (timeSinceVisible < 0.032f)
+            if (timeSinceVisible < 0.032f || float.IsNaN(_lastStepPosX) )

```

Comment:
This looks kind of dirty. 
Why not just use a boolean "isDirty" instead of setting `_lastStepPos`?